doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -2637,15 +2637,14 @@ After the :token:`i_pattern`, a list of binders is allowed.
   .. coqtop:: reset none
 
      From Stdlib Require Import ssreflect.
-     From Stdlib Require Import ZArith Lia.
      Set Implicit Arguments.
      Unset Strict Implicit.
      Unset Printing Implicit Defensive.
 
   .. coqtop:: all
 
      Lemma test : True.
-     have H x (y : nat) : 2 * x + y = x + x + y by lia.
+     have H x (y : nat) : 2 * x + y = x + x + y by admit.
```

Comment:
Littering the documentation with `admit`s and `Axiom`s feels really like a step backward and gives a poor image of Coq. "Why use a proof system where one cannot even prove `2*x = x+x` automatically?" Can't we have the documentation depend on the standard library?

Reply:
Note that there are only 10 such Axiom (over a 56kloc refman). The admit are almost all in the ssreflect chapter that was using lia to demonstrate unrelated features, I could probably improve that. So talking about "littering the documentation" seems quite an overstatement.

Reply:
Even one is a step backwards.

Reply:
I'm unsure of my opinion with respect to the tradeoff between making the refman (in the coq repo) depend on the stdlib (in another repo) vs having a few axioms here and there. If we keep the axioms, they should be hidden from the user (using `.. coqtop:: none`).

Reply:
I think I can get rid of most of tehm, and I will indeed hide any that would remain. I'm working on it.